### DISCUSSION TITLE
Glossary + Acronym

### DISCUSSION THREAD

**Discussion by MariaHenriques** [joaomlourenco/novathesis#469] [Q&A] [answered]
Hello!
I'm trying to add word to the glossary and acronym page simultaneously using the \gls.
When doing so, there is an error saying that the word was previously defined in one of the pages and doesn't appear in both simultaneouly like I wanted.

Any suggestions?
Thank you

**Comment by joaomlourenco** [comment, marked as answer]
You have to use different "keys" in the glossary and acronym definitions. The "contents" may be the same, but the keys have to be different.

**Reply by MariaHenriques** [reply]
Thank you!
